app/code/Magento/QuoteGraphQl/Model/Resolver/SetShippingMethodsOnCart.php

```diff
@@ -85,8 +85,8 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $this->setShippingMethodOnCart->execute(
             $cart,
             $shippingMethod['cart_address_id'],
-            $shippingMethod['shipping_carrier_code'],
```

Comment:
Please note. The shipping methods changes are already applied in a scope of the following PR
https://github.com/magento/graphql-ce/pull/342